txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) (verifyAddress bool) {
```

Comment:
- Caller shouldn't need to pass in `wallet`, can instead access it using `tx.wallet`.
- `verifyAddress` is a verb, verbs should only be used to name functions and methods, use nouns to name variables. I think `addressValid` or `isAddressValid` is a better return variable name.
```suggestion
func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) (isAddressValid bool) {
```